requirements/constraints.txt

```diff
@@ -0,0 +1,16 @@
+# Version constraints for pip-installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why. Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+# Common constraints for edx repos
+-c common_constraints.txt
+
+# The update to pyyaml 6.x failed as docker-compose wants <6,>=3.10. Pinning to <6.0. This constraint will be
+# re-evaluated as part of APER-1556.
```

Comment:
This looks like an old Jira ticket, should this be a referenc to something else? Also, do you need docker-compose?

Reply:
I've removed these lines since we don't need docker-compose